test/e2e/flowaggregator_test.go

```diff
@@ -0,0 +1,416 @@
+    sourceNodeName: k8s-node-master
+    destinationPodName: perftest-b
+    destinationPodNamespace: antrea-test
+    destinationNodeName: k8s-node-master
+    destinationServicePort: 5201
+    destinationServicePortName:
+    ingressNetworkPolicyName: test-flow-aggregator-networkpolicy-ingress
+    ingressNetworkPolicyNamespace: antrea-test
+    egressNetworkPolicyName: test-flow-aggregator-networkpolicy-egress
+    egressNetworkPolicyNamespace: antrea-test
+    destinationClusterIPv4: 0.0.0.0
+    originalExporterIPv4Address: 10.10.0.1
+    originalObservationDomainId: 2134708971
+    octetDeltaCountFromSourceNode: 0
+    octetTotalCountFromSourceNode: 23459802093
+    packetDeltaCountFromSourceNode: 0
+    packetTotalCountFromSourceNode: 537924
+    reverseOctetDeltaCountFromSourceNode: 0
+    reverseOctetTotalCountFromSourceNode: 23108308
+    reversePacketDeltaCountFromSourceNode: 0
+    reversePacketTotalCountFromSourceNode: 444320
+    octetDeltaCountFromDestinationNode: 0
+    octetTotalCountFromDestinationNode: 23459802093
+    packetDeltaCountFromDestinationNode: 0
+    packetTotalCountFromDestinationNode: 537924
+    reverseOctetDeltaCountFromDestinationNode: 0
+    reverseOctetTotalCountFromDestinationNode: 23108308
+    reversePacketDeltaCountFromDestinationNode: 0
+    reversePacketTotalCountFromDestinationNode: 444320
+
+Intra-Node: Flow record information is complete for source and destination e.g. sourcePodName, destinationPodName
+Inter-Node: Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info e.g., sourcePodName, sourcePodNamespace, sourceNodeName etc.
+AntreaProxy enabled (Intra-Node): Flow record information is complete for source and destination along with K8s service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
+AntreaProxy enabled (Inter-Node): Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info like in Inter-Node case along with K8s Service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
+*/
+
+const (
+	ingressNetworkPolicyName = "test-flow-aggregator-networkpolicy-ingress"
+	egressNetworkPolicyName  = "test-flow-aggregator-networkpolicy-egress"
+	collectorCheckTimeout    = 10 * time.Second
+	// Single iperf run results in two connections with separate ports (control connection and actual data connection).
+	// As 5s is export interval and iperf traffic runs for 10s, we expect about 4 records exporting to the flow aggregator.
+	// Since flow aggregator will aggregate records based on 5-tuple connection key, we expect 2 records.
+	expectedNumDataRecords = 2
+)
+
+func TestFlowAggregator(t *testing.T) {
+	// TODO: remove this limitation after flow aggregator supports IPv6
+	skipIfIPv6Cluster(t)
+	skipIfNotIPv4Cluster(t)
+	skipIfProviderIs(t, "remote", "This test is not yet supported in jenkins e2e runs.")
```

Comment:
jenkins is using the "vagrant" provider apparently, as discussed offline

Reply:
removed.